python/tests/transform/camera/test_distortion.py

```diff
@@ -130,3 +131,23 @@ def test_undistort_points():
     with pytest.raises(NotImplementedError):
         new_cam_param, new_points = undistort_points(
             distorted_cam=pytorch3d_cam, points=test_points)
+
+
+def test_fast_undistort_images():
+    fisheye_param = FisheyeCameraParameter(name='distort')
+    fisheye_param.load(os.path.join(input_dir, 'dist_fisheye_param.json'))
+    fisheye_param.set_KRT(R=np.eye(3))
+    intrinsic_backup = np.asarray(fisheye_param.get_intrinsic())
+    test_img = cv2.imread(filename=os.path.join(input_dir, 'dist_img.png'))
+    img_backup = test_img.copy()
+    undistortor = FastImageUndistortor(fisheye_param=fisheye_param)
+    new_img = undistortor.undistort_image(test_img)
```

Comment:
Why not just name it `undist_img`?

Reply:
Fixed in [c43b285](https://github.com/openxrlab/xrprimer/pull/22/commits/c43b285382863fd986b10a8d1fa93aa36ef95e9e).